nelua/cdefs.lua

```diff
@@ -170,6 +170,9 @@ cdefs.target_info_code = [[
 #if defined(__gnu_linux__)
   is_gnu_linux = true;
 #endif
+#if defined(WINAPI_FAMILY_SYSTEM) || defined(WINAPI_PARTITION_APP)
```

Comment:
This seems like it will do nothing, because looks like `WINAPI_FAMILY_SYSTEM` is defined only when including `winapifamily.h`, which will not happen here. This line could be removed.

Reply:
Removed!